Assets/MixedRealityToolkit/Inspectors/MixedRealityToolkitInspector.cs

```diff
@@ -118,11 +143,267 @@ public override void OnInspectorGUI()
+                {
+                    GUI.color = config.SelectedSubjects == 0 ? MixedRealityInspectorUtility.SuccessColor : MixedRealityInspectorUtility.DisabledColor;
+                    if (GUILayout.Button("(None)", EditorStyles.miniButton))
+                    {
+                        config.SelectedSubjects = 0;
+                    }
+
+                    GUI.color = config.SelectedSubjects == SubjectTag.All ? MixedRealityInspectorUtility.SuccessColor : MixedRealityInspectorUtility.DisabledColor;
+                    if (GUILayout.Button("(All)", EditorStyles.miniButton))
+                    {
+                        config.SelectedSubjects = SubjectTag.All;
+                    }
+                }
+
+                List<SubjectTag> sortedTags = new List<SubjectTag>();
+                foreach (SubjectTag tag in Enum.GetValues(typeof(SubjectTag)))
+                {
+                    if (tag == SubjectTag.All)
+                    {
+                        continue;
+                    }
+
+                    sortedTags.Add(tag);
+                }
+
+                sortedTags.Sort(delegate (SubjectTag t1, SubjectTag t2) { return t1.ToString().CompareTo(t2.ToString()); });
+
+                foreach (SubjectTag tag in sortedTags)
+                {
+                    if (numButtons >= maxSubjectButtonsPerLine)
+                    {
+                        numButtons = 0;
+                        EditorGUILayout.BeginHorizontal();
+                        horizontalGroupOpen = true;
+                    }
+
+
+                    bool selected = (config.SelectedSubjects & tag) != 0;
+                    GUI.color = selected ? MixedRealityInspectorUtility.SuccessColor : MixedRealityInspectorUtility.DisabledColor;
+                    if (GUILayout.Button(tag.ToString(), EditorStyles.miniButton))
+                    {
+                        if (selected)
+                        {
+                            config.SelectedSubjects &= ~tag;
+                        }
+                        else
+                        {
+                            config.SelectedSubjects |= tag;
+                        }
+                    }
+                    GUI.color = Color.white;
+
+                    numButtons++;
+                    if (numButtons >= maxSubjectButtonsPerLine)
+                    {
+                        EditorGUILayout.EndHorizontal();
+                        horizontalGroupOpen = false;
+                    }
+                }
+
+                if (horizontalGroupOpen)
+                {
+                    EditorGUILayout.EndHorizontal();
+                }
+            }
+
+            EditorGUILayout.Space();
+            EditorGUILayout.Space();
+
+            EditorGUILayout.HelpBox("Enter keywords below to search for fields.\nIf any subjects are selected, those will be used to narrow your search. Select (None) to search all fields.", MessageType.Info);
```

Comment:
Switch to just bolded title/label "Search Fields" Or "Search Keywords" and if you want to be fancy add a search magnifying glass icon inline with the text box. I think there is InspectorUIUtility or MRStyleUtility?